tests/endpoints/bookstore_grpc/server/server.go

```diff
@@ -34,7 +34,6 @@ import (
 
 	bspbv1 "github.com/GoogleCloudPlatform/esp-v2/tests/endpoints/bookstore_grpc/proto/v1"
 	bspbv2 "github.com/GoogleCloudPlatform/esp-v2/tests/endpoints/bookstore_grpc/proto/v2"
-	healthgrpc "google.golang.org/grpc/health/grpc_health_v1"
```

Comment:
Please revert. This is needed for copybara into google3. gRPC imports need to be different from proto imports